keras/backend/theano_backend.py

```diff
@@ -202,6 +202,12 @@ def any(x, axis=None, keepdims=False):
     return T.any(x, axis=axis, keepdims=keepdims)
 
 
+def all(x, axis=None, keepdims=False):
+    '''Bitwise reduction (logical AND).
+    '''
+    return T.all(x, axis=axis, keepdims=keepdims)
```

Comment:
This is the same as the `any` backend function (Theano returns and int, Tensorflow returns a uint)